CrashLib/Config/CrashLib Release iOS.xcconfig

```diff
@@ -0,0 +1,3 @@
+#include "../../Config/iOS.xcconfig"
+
+ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS)
```

Comment:
Doesn't it require `VALID_ARCHS[sdk=iphoneos*] = $(ARCHS)`?

Reply:
Yes it already does that, note it's including `{root}/Config/iOS.xcconfig` which defines `VALID_ARCHS[sdk=iphoneos*] = $(ARCHS)` (https://github.com/microsoft/appcenter-sdk-apple/blob/fix/arm64e-crash-lib/Config/iOS.xcconfig#L10).

Reply:
My concern is that VALID_ARCHS is overwritten before you change ARCHS here.

Reply:
As soon as it's declared somewhere down the road it works, doesn't need to be redeclared. See the result processed by Xcode:
<img width="702" alt="Screen Shot 2019-05-14 at 11 16 38" src="https://user-images.githubusercontent.com/5318867/57721920-37ed7380-763a-11e9-98ef-f9875bb8e010.png">